### ISSUE TITLE
Expert mode not working again

### ISSUE THREAD

**Issue by SarahSaxifrage** [BiologicalRecordsCentre/plantportal#68]
Hi, sorry @andrewvanbreda but having just gone back in to do some screen recording, the expert mode is now not working again. Also even when creating a normal plot, it is really glitchy. When trying to place a plot the map jumps about a bit and zooms out when clicking on a location and doesn't seem to register when clicking on the map at all at times. Making it all very hard to create a plot in an exact location. @sacrevert

**Comment by andrewvanbreda** [comment]
@SarahSaxifrage 
Acknowledged. 
Will look right now.

**Comment by andrewvanbreda** [comment]
@SarahSaxifrage @sacrevert
I think it is because there are maintenance updates being done on the site which are overwriting code.

@BirenRathod 
We need to be careful of this, as the main Indicia repositories have old forms in them. 
This is because those repositories used to contain project specific forms and I think they are overwriting the changes made on pantheon projects.
Ideally this wouldn't happen if the main Indicia repositories were in perfect condition. I will try to fix this as much as I can (I have done some work to remove old forms previously).
But be aware of this possibility for now.

**Comment by BirenRathod** [comment]
@andrewvanbreda  Thanks for info. Yes, I'm aware that is why I haven't upgraded iform modules. Only upgraded drupal core & its contrib modules.

**Comment by andrewvanbreda** [comment]
@BirenRathod 
Yes that is fine, am just realising on this occasion that isn't the problem. We had that problem before, but it might of been me who did the iForm update.
Just something to note if you do do any iForm stuff.

@sacrevert @SarahSaxifrage 
I am not having this trouble. Oli can you try. Sarah is your internet connection ok?

**Comment by SarahSaxifrage** [comment]
Yes, my connection is great, no issues

**Comment by BirenRathod** [comment]
> Yes that is fine, am just realising on this occasion that isn't the problem. We had that problem before, but it might of been me who did the iForm update. Just something to note if you do do any iForm stuff.
@andrewvanbreda No porblem, I will keep that in mind.

**Comment by andrewvanbreda** [comment]
@SarahSaxifrage 
Did you see this working at all since I did the fix the other day?
Can you give a different browser a go if you have one available?

@sacrevert How is it behaving for you?

**Comment by SarahSaxifrage** [comment]
Yes it did work when I tried the other day, but just now when I was trying to film my screen for training it wasn't

**Comment by SarahSaxifrage** [comment]
I'm using Microsoft edge as had new laptop recently and haven't downloaded anything else yet

**Comment by andrewvanbreda** [comment]
Hi @SarahSaxifrage 
Can you give a different browser a go if you have one available.
Also does it work when it isn't recording the screen?

Nothing has been changed since the other day.

**Comment by SarahSaxifrage** [comment]
No it doesn't work when not screen recording

**Comment by SarahSaxifrage** [comment]
I can't try a new browser as would need to download another one and don't have time today

**Comment by SarahSaxifrage** [comment]
I've managed to try it with Chrome and it seems ok now

**Comment by andrewvanbreda** [comment]
Hi @SarahSaxifrage 
Ok leave this issue open for now, and let me know if it stops working again.

Going by the description, it is probably not a code problem, but something else causing issue..

**Comment by SarahSaxifrage** [comment]
Ok thanks

**Comment by sacrevert** [comment]
@andrewvanbreda there still seems to be something wrong with the drawing tools on the create plot pages, at least in comparison with the NPMS form. At the moment both the draw plot and move map tools are missing from the top right hand corner of the map view where they appear in NPMS. This applies to both expert and normal mode. I'm using Chrome, and the equivalent NPMS page works fine.

**Comment by andrewvanbreda** [comment]
Hi @sacrevert 
Am not getting this problem.
Can you send me 2 things please

1. The URL address of the page including the parameters that appear after the "?" character.
2. A screenshot of what you are seeing.
Cheers.

**Comment by sacrevert** [comment]
Probably i should have specified that it's the TTI page specifically, sorry:
https://plantportal.ceh.ac.uk/tti-create-plot
![image](https://user-images.githubusercontent.com/5985134/181225057-f43f0d6b-9061-483b-8f66-c029ec667c36.png)

**Comment by andrewvanbreda** [comment]
Hi @sacrevert 
I don't think this project ever worked liked that.
You originally just selected the Square and it just auto-zooms into the location, then you are free to click.

However I think what has happened though is that the page has got a little tangled when Expert Mode was added to general Plant Portal and it has interfered with this page.
So what I have done is I have enabled the extra tools for TTI.
I think they should work ok, but I have not texted the page in any detail. I am noticing one unusual thing is that the Width and Length numbers for the plot are exposed in 2 textboxes on the page to the user. Although it probably doesn't do any actually harm other than the plot width/length can be made to be inconsistent with the plot type, but this is probably a new thing that wasn't originally wrong.
Another change that has come across from NPMS is plots can now be rotated.

Note there is no support money available for dealing with tracking The Impact issues at the moment.
